### PR TITLE
[DO NOT MERGE] Remove host routing

### PR DESCRIPTION
**Pull request by tamas-molnar**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -17,11 +17,11 @@ EXPOSE 8080 8081
...
-     -Ddw.methodeContentPlaceholderMapper.endpointConfiguration.primaryNodes=$VULCAN_HOST \
-     -Ddw.documentStoreApi.endpointConfiguration.primaryNodes=$VULCAN_HOST \
-     -Ddw.concordanceApi.endpointConfiguration.primaryNodes=$VULCAN_HOST \
-     -Ddw.consumer.messageConsumer.queueProxyHost=http://$VULCAN_HOST \
-     -Ddw.producer.messageProducer.proxyHostAndPort=$VULCAN_HOST \
+     -Ddw.methodeArticleMapper.endpointConfiguration.primaryNodes=$METHODE_ARTICLE_MAPPER_URL \
+     -Ddw.methodeContentPlaceholderMapper.endpointConfiguration.primaryNodes=$METHODE_CPH_MAPPER_URL \
+     -Ddw.documentStoreApi.endpointConfiguration.primaryNodes=$DOC_STORE_API_URL \
+     -Ddw.concordanceApi.endpointConfiguration.primaryNodes=$CONCORDANCE_API_URL \
+     -Ddw.consumer.messageConsumer.queueProxyHost=http://$KAFKA_PROXY_URL \
```
Comment (kovacshuni) [inline Dockerfile]: Producer and consumer have different names for the same kind of parameter, also one has http in front of the value and the other doesn't. Please arrange

Comment (bogdanguranda) [reply Dockerfile]: I'll adjust them. I'll check if the consumer/producer is a inconsistency, I see it in every mapper repo though.

Comment (bogdanguranda) [reply Dockerfile]: So the names are defined in a 3rd party library, so only by doing some wrapping class would work, but I don't think it's worth it.

Comment (kovacshuni) [reply Dockerfile]: That's ok, leave it.

Comment (kovacshuni) [reply Dockerfile]: Thanks #http

**Comment by bogdanguranda** [comment]
Closing this as it was included in the following PR: https://github.com/Financial-Times/methode-article-internal-components-mapper/pull/25

**Pull request closed without merging**